scripts/globals/settings.lua

```diff
@@ -104,14 +104,13 @@ RELIC_3RD_UPGRADE_WAIT_TIME = 3600;     -- wait time for 3rd relic upgrade (stag
 FREE_COP_DYNAMIS = 1 ; -- Authorize player to entering inside COP Dynamis without completing COP mission ( 1 = enable 0= disable)
 
 -- QUEST/MISSION SPECIFIC SETTINGS
-WSNM_LEVEL = 70; -- Min Level to get WSNM Quests
-WSNM_SKILL_LEVEL = 240;
 AF1_QUEST_LEVEL = 40; -- Minimum level to start AF1 quest
 AF2_QUEST_LEVEL = 50; -- Minimum level to start AF2 quest
 AF3_QUEST_LEVEL = 50; -- Minimum level to start AF3 quest
 AF1_FAME = 20; -- base fame for completing an AF1 quest
 AF2_FAME = 40; -- base fame for completing an AF2 quest
 AF3_FAME = 60; -- base fame for completing an AF3 quest
+WSNM_FAME = 60; -- base fame for completing a Weapon Skill quest
```

Comment:
I'm just saying skill level setting is redundant because you can easy change the skill lv in the db.

as far as the level I was saying lv for-quest sounded more useful than a setting for the fame reward.

they can all 3 get nuked.

Reply:
the AF fame settings make me wanna facepalm too really, but they already exist so they can stay for now, just suggesting we don't add it it

Reply:
Fair. Removed.